main.js

```diff
@@ -1,9 +1,30 @@
-for(let i = 1; i < 100; i++){
-  if (i % 15 === 0){
-    console.log(`FizzBuzz ${i}`);
-   } else if (i % 5 === 0){
-    console.log(`Buzz ${i}`);
-   } else if (i % 3 === 0){
-    console.log(`Fizz ${i}`);
- }
-}
\ No newline at end of file
+"use strict";
+console.log("=========================");
+console.log("現在持っているタスク一覧");
+console.log("=========================");
+const list = [
+  {task:"机を片付ける",genre:"掃除"},
+  {task:"牛乳を買う",genre:"買い物"},
+  {task:"散歩する",genre:"運動"},
+];
+  for (let i = 0; i < list.length; i++ ){
+    console.log(`${i} : [内容] ${list[i].task}、[ジャンル] ${list[i].genre}`);
+}
+var c = list.length;
+var a = prompt("タスクを入力してください");
+var b = prompt("ジャンルを入力してください");
+  if(a && b){
+    list[c] = {task:a,genre:b};
+  }
```

Comment:
- `var` は昨今ではほとんど使われないので `let` または `const` を使いましょう
- 変数名は「何が入っているか」一見してわかる名前である方が好ましいです
- 配列への要素の追加は `push` という便利なものがあります

```suggestion
const task = prompt("タスクを入力してください");
const genre = prompt("ジャンルを入力してください");
list.push({task: task, genre: genre})
```